features/steps/partcad-cli/commands/status.py

```
@@ -0,0 +1,17 @@
+import os
+import yaml
+from behave import given, when, then
+
+
+@then('a file named "{filename}" should be created with content')
+def step_impl(context, filename):
```

Ruff (0.8.0)

Missing return type annotation for public function `step_impl`

Add return type annotation: `None`

(ANN201)